### ISSUE TITLE
Provide smesher geo data

### ISSUE THREAD

**Issue by lrettig** [spacemeshos/api#20]
team spacemesh will need to provide an API end point that will return a list of geo locations (lat/long) as well as the allocated post size in that location for each location.

from https://spacemesh.slack.com/archives/CREMHKGT0/p1590064728042600

**Comment by lrettig** [comment]
This is something of a special case. As @avive pointed out, these data aren't part of the mesh or the meta-mesh (state), so we need a new endpoint that's part of a separate API, probably part of the telemetry backend. Strictly speaking, therefore, this issue doesn't belong in this repo, but I'm not sure what's the right place for it - maybe Spacecraft?

How's this for the proto?

```
message NodeGeo {
    uint32 lat;
    uint32 lon;
    uint64 space;
}

message NodeGeos {
    repeated NodeGeo node_geos = 1;
}

rpc NodeGeos(google.protobuf.Empty) returns (NodeGeos);
```

**Comment by avive** [comment]
- We can just return a stream of NodeGeo - one for each one we know about in this or pervious epoch. No need for NodeGeos
- This should ideally be part of telemetry client api.

**Comment by avive** [comment]
after thinking about this, I think that this should be a seperate api as the data is collected by an entity that is not a full node so it doesn't make sense to add it here as the services in this repo are supposed to be provided by full nodes only for now. Maybe in a new project - telematry client api?

**Comment by lrettig** [comment]
Agree, I pointed out the same thing in the initial comment above :) Do we have a "telemetry client" though?

**Comment by avive** [comment]
Isn't telemetry client part of the full node telemetry task?

**Comment by lrettig** [comment]
> We are planning to collect telemetry from nodes and going to have this in a centralized data base

How does one query this "centralized database"? That's what I'm referring to. Are we on the same page?

**Comment by avive** [comment]
yes, the telemetry backend will need to provide this api but I think this should be a go-spacemesh issue as part of the telemetry task and not related to the api.

**Comment by lrettig** [comment]
I'm removing this from the API project as it's not really part of the API as speced now, but I'll leave this issue open to track it until it has a better home.
